modules/jaeger-deployment-best-practices.adoc

```diff
@@ -0,0 +1,13 @@
+////
+This module included in the following assemblies:
+*  /jaeger/jaeger_install/rhbjaeger-deploying.adoc
+////
+
+[id="jager-deployment-best-practices_{context}"]
+= Deployment best practices
+:pantheon-module-type: CONCEPT
+
+
+* Jaeger instance names should be unique. If you want to have multiple Jaeger instances and are using sidecar injected Jaeger agents, then the Jaeger instances should have unique names, and the injection annotation should explicitly specify the Jaeger instance name the tracing data should be reported to.
+
+* If you are installing Jaeger as part of Red Hat OpenShift Service Mesh, Jaeger should be installed in the same namespace as Service Mesh.  That is, the `ServiceMeshControlPlane` and Jaeger resources must in the same namespace.
```

Comment:
I think if you combine both of these sentences instead of clarifying the first with the second, it would be more effective. 

So:

>If you are installing Jaeger as part of Red Hat OpenShift Service Mesh, Jaeger resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource.